### PR TITLE
PAYARA-2905 Prevent creation of databases used for Timer Service when only non-persistent timers are present

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

**Comment by jGauravGupta** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
jenkins test please

File: appserver/ejb/ejb-timer-service-app/src/main/java/com/sun/ejb/timers/TimerWelcomeServlet.java [hunk 1]
```diff
@@ -88,7 +85,15 @@ protected void processRequest(HttpServletRequest request, HttpServletResponse re
             Set persistenttimers = timer.findActiveTimersOwnedByThisServer();
             // Non-persistent timers get directly from the service
 
-            EJBTimerService ejbTimerService = EJBTimerService.getEJBTimerService();
+            NonPersistentEJBTimerService ejbTimerService = null;
+            if (EJBTimerService.isPersistentTimerServiceLoaded()) {
+                ejbTimerService = (NonPersistentEJBTimerService) EJBTimerService.getPersistentTimerService();
+            } else if (EJBTimerService.isPersistentTimerServiceLoaded()) {
+                ejbTimerService = (NonPersistentEJBTimerService) EJBTimerService.getNonPersistentTimerService();
+            }
```
Comment (svendiedrichsen) [inline appserver/ejb/ejb-timer-service-app/src/main/java/com/sun/ejb/timers/TimerWelcomeServlet.java]: This if/else statement irritates me. Is this really meant this way?

**Comment by jGauravGupta** [comment]
jenkins test please

File: appserver/ejb/ejb-timer-service-app/src/main/java/com/sun/ejb/timers/TimerWelcomeServlet.java (hunk 1 above)
Comment (jGauravGupta) [reply appserver/ejb/ejb-timer-service-app/src/main/java/com/sun/ejb/timers/TimerWelcomeServlet.java]: That's a typo. Thanks for pointing out.

**Comment by jGauravGupta** [comment]
jenkins test please

**Pull request merged**